src/doc/building_visit/Linux_X11_Qt.rst

```diff
@@ -9,13 +9,10 @@ Many developers encounter errors building Qt due to missing support libraries re
 Here are a couple of Qt links that prove helpful in knowing which packages need to be installed for particular OS distros.
 
 * For Qt 5
-
-  * The libxcb section of `Building Qt5 from Git <https://wiki.qt.io/Building_Qt_5_from_Git>`_
-
-  * Qt5's `X11 Requirements <https://doc.qt.io/qt-5/linux-requirements.html>`_ page.
+  * The libxcb section of `Building Qt5 from Git <https://wiki.qt.io/Building_Qt_5_from_Git>`_  
+  * Qt5's `X11 Requirements <https://doc.qt.io/qt-5/linux-requirements.html>`__ page.
 
 * For Qt 6 (only on develop branch)
-
   * Qt6's `X11 Requirements <https://doc.qt.io/qt-6/linux-requirements.html>`__ page.
```

Comment:
```suggestion

  * Qt6's `X11 Requirements <https://doc.qt.io/qt-6/linux-requirements.html>`__ page.
```

Reply:
Am rendering locally too and getting warnings related to this. I'll push an update momentary.

Reply:
What is the warning? I didn't see any warnings when I originally added the blank lines.

Reply:
Also, if warnings are treated as errors on RTD, why didn't the build fail?